OrbitBase/include/OrbitBase/SafeStrerror.h

```diff
@@ -0,0 +1,6 @@
+#ifndef ORBIT_BASE_SAFE_STRERROR_H_
+#define ORBIT_BASE_SAFE_STRERROR_H_
+
+char* SafeStrerror(int errnum);
```

Comment:
Please add a comment why this exists and why this should be preferred over strerror.

Reply:
Done.